session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -68,7 +68,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class Session {
+public class Session extends AsyncSession {
```

Comment:
Session extends AsyncSession violate the LSP. Consider to use combination rather than inherit